AnimalMatcher/AnimalMatcher.Data/Models/Message.cs

```diff
@@ -0,0 +1,27 @@
+﻿namespace AnimalMatcher.Data.Models
+{
+    using AnimalMatcher.Common.Constants;
+    using System.ComponentModel.DataAnnotations;
+
+    public class Message
+    {
+        public int Id { get; set; }
+
+        [Required]
+        public int RecipientId { get; set; }
+
+        public Pet Recipient { get; set; }
+
+        [Required]
+        public int SenderId { get; set; }
+
+        public Pet Sender { get; set; }
+
+        [MaxLength(MessageConstants.SubjectMaxLength)]
+        public string MessageSubject { get; set; }
```

Comment:
Messages will not need a Subject - as we plan to have a chat between two pets which will contain a list of messages between them and will have only a message body and a timestamp - when the message was sent

Reply:
Removed subject now.